create name badge

depial opened the pull request:
A possible fix for Issue #934 

- exchanged the more specific word `print` for the more general `create` throughout.
- tried to implement backwards compatibility with old solutions, with `exemplar.jl` left with the old function name `print_name_badge` as a test.

colinleach commented:
Thanks for the attempt, but...

As discussed in #934, I'm reluctant to change the code. Altering task titles is less disruptive, and I'm fine with that.

depial commented:
I've gone ahead and reverted the function name change and took out the backwards compatibility fix. 

I've tried to change the word `print` to `create` in all other contexts and added some clarity that the badges are worn in the first mention of them. If we decide to proceed, please give a good look through as I caught a couple of `print`s I missed on the first go around.
